tests/e2e/lib/backup.go

```diff
@@ -12,7 +16,55 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func CreateBackupForNamespaces(ocClient client.Client, veleroNamespace, backupName string, namespaces []string, defaultVolumesToFsBackup bool, snapshotMoveData bool) (velero.Backup, error) {
+// ParseDuration parses a string representation of a duration with suffixes (s, m, h)
+// and converts it to the k8s.io/apimachinery/pkg/apis/meta/v1.Duration type.
+//
+// The input string should have the format "<value><suffix>", where:
+// - <value> is a positive integer representing the duration value.
+// - <suffix> is a character representing the duration unit:
+//   - "s" for seconds
+//   - "m" for minutes
+//   - "h" for hours
+//
+// Example valid inputs: "10s", "5m", "1h"
+//
+// Returns the parsed duration as a v1.Duration object, or an error if parsing fails.
+func parseDuration(durationStr string) (metav1.Duration, error) {
+	durationStr = strings.TrimSpace(durationStr)
+	suffix := durationStr[len(durationStr)-1:]
+	valueStr := durationStr[:len(durationStr)-1]
+
+	value, err := strconv.Atoi(valueStr)
+	if err != nil {
+		return metav1.Duration{}, errors.New("Invalid duration value: " + valueStr)
+	}
+
+	var dur time.Duration
+	switch suffix {
+	case "s":
+		dur = time.Duration(value) * time.Second
+	case "m":
+		dur = time.Duration(value) * time.Minute
+	case "h":
+		dur = time.Duration(value) * time.Hour
+	default:
+		return metav1.Duration{}, errors.New("Invalid duration suffix: " + suffix)
+	}
+
+	return metav1.Duration{Duration: dur}, nil
+}
```

Comment:
If you're importing time package, just use it's parse function no?
```suggestion
// ParseDuration parses a duration string.
// A duration string is a possibly signed sequence of
// decimal numbers, each with optional fraction and a unit suffix,
// such as "300ms", "-1.5h" or "2h45m".
// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
func parseDuration(durationStr string) (metav1.Duration, error) {
	time, err := time.ParseDuration(durationStr)
	return metav1.Duration{Duration: time}, err
}
```